radio/src/tasks.cpp

```diff
@@ -41,6 +41,11 @@ void stackPaint()
 #if defined(CLI)
   cliStack.paint();
 #endif
+//OW
+#if defined(TELEMETRY_MAVLINK)
```

Comment:
Oho! You added a new task?

Reply:
yeah, the stuff needs to be done somewhere, and the mixer task is too frequent and too tight already, and the other tasks way too low rate

Reply:
Ok, understood!